content/02.main-text.md

```diff
@@ -7,7 +7,7 @@ Open research – which includes sharing code, data, and manuscripts – benefit
 Here we describe the benefits of writing review articles openly, where the planning, organizing, writing, and editing occur collaboratively in a public forum where participants are free to join as they wish.
 Reviews presenting the state of the art in a scientific field are often prepared by a single research group or a small team of colleagues.
 In contrast, broadly opening the writing process to anyone engaged in the topic can help maximize the review's value by facilitating the representation of diverse opinions and the broad coverage of relevant research.
-Review authors can engage with the authors of original research to clarify their methods and results and present them accurately (for example, [@url:https://github.com/greenelab/deep-review/issues/213]).
+Review authors can engage with the authors of original research to clarify their methods and results and present them accurately (for example, @url:https://github.com/greenelab/deep-review/issues/213).
```

Comment:
From the [pandoc docs](http://pandoc.org/MANUAL.html#citation-rendering)

> Citations go inside square brackets and are separated by semicolons: `Blah blah [@smith04; @doe99].`

> You can also write an in-text citation, as follows: `@smith04 says blah.`

How in-text citations are rendered depends on the selected CSL style, I believe.